uber/models.py

```diff
@@ -1938,6 +1953,22 @@ class ArbitraryCharge(MagModel):
     _repr_attr_names = ['what']
 
 
+class LedgerItem(MagModel):
+    ident = Column(UnicodeText, nullable=True)
+    type = Column(Choice(c.LEDGER_TYPE_OPTS), default=c.ACCOUNTS_RECEIVED)
```

Comment:
Whoops, I didn't expect this to be highlighted. Maybe naming this Column `type` isn't a good call?